docs/rule_config.md

```diff
@@ -0,0 +1,51 @@
+Allows to overwrite the default behaviour of the rule changing rule mode and risk value. It accepts a more granular 
+configuration using the filter.
+ 
+{{ inline_source('cfripper.config.rule_config.RuleConfig') }}
+
+## Filters
+
+Allows to set custom rule configuration to matching coincidences. When adding a failure it will check if there is a 
```

Comment:
I would remove the first sentence probably, just start with `When adding a failure`

Also, it is for failure and warning I think